README.md

```diff
@@ -16,8 +16,8 @@ This theme is named after Canadian media theorist Marshall McLuhan, who coined t
 ## Requirements
 
 * PHP >= 7.3
-* WordPress >= 5.5.3
-* Pressbooks >= 5.18.0
+* WordPress >= 5.6.2
+* Pressbooks >= 5.20.0
```

Comment:
Do we need to bump required versions of WP and Pressbooks with each release? Honest question.

Reply:
The wordpress version doesn't need to be bumped in this case. For Pressbooks, if the theme relies on a certain version pressbooks then it should be bumped. In this case, I wasn't 100% if it needs the latest pb version so i bumped anyways.
This should not affect the theme in anyway I believe since the readme.md does not follow the requirements from wordpress.

https://make.wordpress.org/themes/handbook/review/required/#readme-txt-file